recipe/rsync.php

```diff
@@ -137,8 +137,8 @@
     $server = $server->getConfiguration();
     $host = $server->getHost();
     $port = $server->getPort() ? ' -p' . $server->getPort() : '';
-    $identityMethod = $server->getPemFile() ? 'getPemFile' : $server->getPrivateKey() ? 'getPrivateKey' : false;
```

Comment:
Oops, My mistake.

`$identityMethod = $server->getPemFile() ? 'getPemFile' : ($server->getPrivateKey() ? 'getPrivateKey' : false);`

Should fix the problem but if you have a nicer solution feel free to use that.